### ISSUE TITLE
Unet only for classification?

### ISSUE THREAD

**Issue by 9B8DY6** [milesial/Pytorch-UNet#206]
model = UNet(
        in_channels=1,
        out_classes=1,
        dimensions=3,
        num_encoding_blocks=2,
        out_channels_first_layer=8,
        normalization='batch',
        upsampling_type='linear',
        padding=True,
        activation='ReLU',
    )

I want to use UNet for age regression with 3D brain MRI, so I define hyperparamter like beyond. Is it right?

**Comment by milesial** [comment]
For regression you would have to modify the loss to MSE or something else than BCE, and remove the sigmoid/softmax when predicting. You can leave all the other architecture detail as default.
